app/main.py

```diff
@@ -2,0 +1,42 @@
+from types import TracebackType
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name: str = name
+        self.health: int = health
+        self.hidden: bool = False
+        Animal.alive.append(self)
+
+    def __enter__(self) -> None:
+        pass
+
+    def __exit__(
+            self,
+            exc_type: type,
+            exc_val: Exception,
+            exc_tb: TracebackType
+    ) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, "
+                f"Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
```

Comment:
Use string interpolation instead of concatenation, as per the checklist. So, instead of concatenating strings with '+', use f-strings or the `format` method. This is a code style issue.